x/auth/types/txbuilder.go

```diff
@@ -74,6 +76,71 @@ func NewTxBuilderFromCLI(input io.Reader) TxBuilder {
 	return txbldr
 }
 
+// NewTxBuilderFromCLI returns a new initialized TxBuilder with parameters extracted from
+// FlagSet (It should deprecate NewTxBuilderFromCLI).
```

Comment:
Add a deprecated note to `NewTxBuilderFromCLI`. Also, remember we'll end up renaming this too for consistency.